reana-ui/src/util.js

```diff
@@ -230,6 +230,11 @@ export function getMimeType(fileName) {
   if (WHITELIST.find((ext) => fileName.endsWith(ext))) {
     return "text/plain";
   }
+  if (fileName.endsWith(".root")) {
+    // ROOT does not have a registered MIME-type yet
+    // See https://github.com/root-project/root/issues/6771
+    return "application/x-root";
+  }
```

Comment:
What do you think about using the [`define`](https://github.com/broofa/mime#mimedefinetypemap-force--false) method from the `mime` module? We could also use it to define in a flexible way further non-standard mappings (for example, `.sh` files are not previewed by REANA)
```javascript
mime.define(
    {
      "application/x-root": ["root"],
      "text/x-shellscript": ["sh"],
    },
    true
  );
```

Reply:
Done! I have created a new `Mime` local object, so that we do not modify the global `mime` variable provided by the library